src/common/utils/fqdn.cpp

```diff
@@ -23,36 +23,51 @@
 #include "logger.hpp"
 #include <boost/asio.hpp>
 #include <iostream>
+#include <chrono>
+#include <thread>
+
+#define MAX_NB_RESOLVE_TRIES 4
+#define TIME_BETWEEN_TRIES 2
```

Comment:
I would recommend a time unit indication (second or milliseconds for example)

Reply:
`#define TIME_IN_SECS_BETWEEN_TRIES 2` ?

Reply:
yes for example

Reply:
so the reader don't have to search for unit in the code